modules/customize-certificates-replace-default-router.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/replacing-default-ingress-certificate.adoc
+
+[id="replacing-default-ingress-{context}"]
+= Replacing the default ingress certificate
+
+You can replace the default ingress certificate for all
+applications under the `.apps` subdomain. This will update
+the certificate used for the web console and CLI.
+
+.Prerequisites
+
+* You must have a wildcard certificate for use. 
```

Comment:
cert/key in PEM format.
A wildcard certificate that is issued for (SAN) for *.apps.<domain>.

Reply:
These prerequisites have been included.